src/transformers/testing_utils.py

```diff
@@ -654,21 +655,31 @@ def get_env(self):
         env["PYTHONPATH"] = ":".join(paths)
         return env
 
-    def get_auto_remove_tmp_dir(self, tmp_dir=None, after=True, before=False):
+    def get_auto_remove_tmp_dir(self, tmp_dir=None, before=None, after=None):
         """
         Args:
             tmp_dir (:obj:`string`, `optional`):
-                use this path, if None a unique path will be assigned
-            before (:obj:`bool`, `optional`, defaults to :obj:`False`):
-                if `True` and tmp dir already exists make sure to empty it right away
-            after (:obj:`bool`, `optional`, defaults to :obj:`True`):
-                delete the tmp dir at the end of the test
+                if not :obj:`None` use this path, otherwise, use a unique tmp path will be created
+            before (:obj:`bool`, `optional`, defaults to :obj:`None`):
```

Comment:
```suggestion
            before (:obj:`bool`, `optional`):
```
We don't indicate defaults to None for optional things.